usmqe_tests/web/tendrl/test_login.py

```diff
@@ -0,0 +1,89 @@
+"""
+Description: Simple tests for log in on tendrl UI
+
+Author: ltrilety
+"""
+
+
+import pytest
+from selenium.webdriver.common.keys import Keys
+
+from webstr.selenium.ui.support import WaitForWebstrPage
+from usmqe.web.tendrl.loginpage import pages as loginpage
+from usmqe.web.tendrl.mainpage.landing_page.pages import get_landing_page
+
+LOGGER = pytest.get_logger('login_test', module=True)
+
+
+def test_positive_login(testcase_set, testcase_end):
```

Comment:
First of all, it was split from the beginning. That said I wonder why the interest now.
Anyway there's a really easy explanation behind that:
1. It's easy to read  and even when `testcase_end` fixture requires `testcase_set`, `testcase_set` is called only once.
2. `testcase_set` exist as individual, because there could be a test which doesn't want to close a browser. Well I admit probably never will, but it could.